### ISSUE TITLE
Composer download error

### ISSUE THREAD

**Issue by IvanZugec** [phenaproxima/starshot-prototype#40]
When I run:
```
composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev
```

I get: 
```
Loading composer repositories with package information

In CurlDownloader.php line 640:

  The "https://api.github.com/repos/phenaproxima/starshot-core" file could not be downloaded (HTTP/2 404 ):
  {"message":"Not Found","documentation_url":"https://docs.github.com/rest/repos/repos#get-a-repository"}


install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--download-only] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [--audit] [--audit-format AUDIT-FORMAT] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]
```

Anyone else have this issue?

**Comment by phenaproxima** [comment]
Ooof, good catch. That was an oversight on my part this morning. It should be fixed now (in ea9474ab9aa770439f1e3347851db6b0aef2e329). Can you try again?

**Comment by IvanZugec** [comment]
Thanks @phenaproxima, that fixed it.
